snakemake/remote/zenodo.py

```diff
@@ -99,11 +99,12 @@ def download(self):
             )
 
     def upload(self):
-        if self.size() <= 100000000:
-            self._zen.upload(self.local_file(), self.remote_file())
-        else:
-            raise ZenodoFileException(
-                "Current Zenodo stable API supports <=100MB per file."
+        with open(self.local_file(), "rb") as lf:
+            self._zen._api_request(
+                self._zen.bucket
+                + "/{}".format(os.path.basename(self.remote_file())),
+                method="PUT",
+                data=lf
```

Comment:
No 100Mb limit anymore: updated upload method to new Zenodo files API with a PUT request to a 'bucket' link.